packages/experimental/ActivityIndicator/src/ActivityIndicator.styling.tsx

```diff
@@ -0,0 +1,55 @@
+import { UseStylingOptions, buildProps } from '@fluentui-react-native/framework';
+import { Appearance } from 'react-native';
+import {
+  activityIndicatorName,
+  ActivityIndicatorProps,
+  ActivityIndicatorSlotProps,
+  ActivityIndicatorTokens,
+} from './ActivityIndicator.types';
+
+export const diameterSizeMap: { [key: string]: number } = {
+  xSmall: 12,
+  small: 16,
+  medium: 24,
+  large: 32,
+  xLarge: 36,
+};
+export const lineThicknessSizeMap: { [key: string]: number } = {
+  xSmall: 1,
+  small: 1,
+  medium: 2,
+  large: 3,
+  xLarge: 4,
+};
+
+export const stylingSettings: UseStylingOptions<ActivityIndicatorProps, ActivityIndicatorSlotProps, ActivityIndicatorTokens> = {
+  tokens: [
+    () => ({
+      activityIndicatorColor: Appearance.getColorScheme() === 'light' ? '#BDBDBD' : '#666666',
+      lineThickness: 'medium',
+      size: 'medium',
+    }),
+    activityIndicatorName,
+  ],
+  tokensThatAreAlsoProps: 'all',
+  slotProps: {
+    root: buildProps(
+      (tokens: ActivityIndicatorTokens) => ({
+        activityIndicatorColor: tokens.activityIndicatorColor,
+        size: diameterSizeMap[tokens.size],
+        lineThickness: tokens.lineThickness != 'medium' ? lineThicknessSizeMap[tokens.lineThickness] : lineThicknessSizeMap[tokens.size],
+        accessibilityRole: 'progressbar',
+        accessible: true,
+        style: { width: diameterSizeMap[tokens.size] },
```

Comment:
You don't need height here too?

Reply:
The View automatically adjusts to the height of the ActivityIndicator, but the width is automatically the width of the stack. Setting the height would be fine, as a just in case.

Reply:
The width/height difference there might be a side effect of flexbox and the <Stack> element that ActivityIndicator is hosted in. I think we should set both. Thanks for the clarification though =)